lib/dialog/style.scss

```diff
@@ -39,6 +39,7 @@
   display: flex;
   flex-direction: column;
   flex: 1 0 auto;
+  max-height: calc(100vh - 56px - 20px);
```

Comment:
The max height of the content view is calculated by subtracting the header height (`56px`) and some padding (`20px`) to the window height (`100vh`).

Reply:
Thanks so much for diving in and starting on fixes!
There are a couple of small things. In a [previous PR](https://github.com/Automattic/simplenote-electron/pull/2685#issuecomment-797630876), I was working on one dialog in particular and took a similar approach to what you did here to set the max height. It was decided instead that we could use a height of 420px as the max instead. 
The dialog in your video works well as one of the inner elements of the dialog has an overflow set for it. For other dialogs this can cause the content to overflow out of it. Would be good to add an `overflow: auto;` to this as well.
Not sure how far you want to go with this but #2777 is sort of related to this as well. 
The plan was to have the entire height of the dialog be a max of 420px, then have the inner content be scrollable if needed. The header, and if there were any action buttons with the dialog at the bottom, would remain static and always visible without scrolling.

Reply:
>There are a couple of small things. In a previous PR, I was working on one dialog in particular and took a similar approach to what you did here to set the max height. It was decided instead that we could use a height of 420px as the max instead.

I just checked that the [minimum window is set to `520px`](https://github.com/Automattic/simplenote-electron/blob/f8165a48c35423e6bc83513ad7cf9bd77f9e7aea/desktop/app.js#L67:L67) so `420px` should be enough. However the content is still cut off if you zoom in the UI (see attached video), as far as I checked the only way to prevent it is by using the viewport height with `vh` unit 😞 .

https://user-images.githubusercontent.com/14905380/112873218-a36f4b80-90c1-11eb-87d2-566b2b57302d.mp4

>The dialog in your video works well as one of the inner elements of the dialog has an overflow set for it. For other dialogs this can cause the content to overflow out of it. Would be good to add an overflow: auto; to this as well.

Good point! I think it would be a great idea to add overflow to the content view, this way we assure that it's always scrollable. 

Another thing it would be interesting is to review the dialogs and set overflow to the sections that would require it. As an example, I'm working on this [issue](https://github.com/Automattic/simplenote-electron/issues/2716) related `UnsynchronizedConfirmation` dialog and here I think makes more sense to make the changes list scrollable not the whole content (check follow video).

https://user-images.githubusercontent.com/14905380/112873709-47f18d80-90c2-11eb-85ad-217e794416bf.mp4

>Not sure how far you want to go with this but #2777 is sort of related to this as well.

I'm still getting familiar with the project but I can take a look if you want.

Reply:
One option I'm considering, although I'm not sure about [potential compatibility issues](https://chromestatus.com/feature/5714277878988800#details), is to use this for calculating the max height:
`max-height: unquote('min(100vh - 20px, 420px)');`.

This will allow us to set the maximum height to `420px` but if for any reason the window gets smaller (looks like this is what's happening when zooming in), we use the window height to prevent the cut off.

Note the use of `unquote` for prevent using the Sass's `min` function because it doesn't support `vh` units.

Reply:
Using the min function seems like a good approach to me here to get the 420px height, but also ensure it works on zoomed in views.

My thoughts on #2777 were very similar to what you're thinking about for the `UnsynchronizedConfirmation` dialog.

Reply:
After a second thought, I decided to use media queries instead of this approach because it looks way safer ([related commit](https://github.com/Automattic/simplenote-electron/pull/2815/commits/b91a2f115890225a1193997570fc62ac5bad54a5)).